ui/src/app/base/selectors/pod/pod.test.js

```diff
@@ -77,4 +77,46 @@ describe("pod selectors", () => {
       id: 222,
     });
   });
+
+  it("can get a pod's host machine", () => {
+    const state = {
+      controller: {
+        items: [],
```

Comment:
Would it be useful to add a controller here, so that the test also validates that a machine is returned by default?